client/my-sites/marketing/traffic/index.js

```diff
@@ -108,7 +109,8 @@ const connectComponent = connect( ( state ) => {
 	const siteId = getSelectedSiteId( state );
 	const isAdmin = canCurrentUser( state, siteId, 'manage_options' );
 	const isJetpack = isJetpackSite( state, siteId );
-	const isJetpackAdmin = isJetpack && isAdmin;
+	const isFreeAtomic = isFreeAtomicSite( state, siteId );
+	const isJetpackAdmin = isJetpack && isAdmin && ! isFreeAtomic;
```

Comment:
I used `isFreeAtomic` rather than `isAtomic` here since these settings are currently available to Atomic sites.